docs/documentation/securing_apps/topics/oidc/fapi-support.adoc

```diff
@@ -36,6 +36,12 @@ in the more strict way to enforce some of the requirements. Especially:
 * If your client does not use PAR, make sure that it uses encrypted OIDC request objects. This can be achieved by using a client profile with the `secure-request-object` executor configured with `Encryption Required` enabled.
 * Make sure that for JWS, the client uses the `PS256` algorithm. For JWE, the client should use the `RSA-OAEP` with `A256GCM`. This may need to be set in all the link:{adminguide_link}#_oidc_clients[Client Settings] where these algorithms are applicable.
 
+==== Australia Consumer Data Right (CDR) Security Profile
+
+{project_name} is compliant with the https://consumerdatastandardsaustralia.github.io/standards/#security-profile[Australia Consumer Data Right Security Profile].
+
+If you want to apply Australia CDR security profile, you need to use `fapi-1-advanced` profile because Australia CDR security profile is based on FAPI 1.0 Advanced security profile. If your client also applies PAR, make sure that it applies RFC 7637 Proof Key for Code Exchange (PKCE) because Australia CDR security profile requires to apply PKCE when applying PAR. This can be achieved by using a client profile with the `pkce-enforcer` executor.
```

Comment:
```suggestion
If you want to apply the Australia CDR security profile, you need to use `fapi-1-advanced` profile because the Australia CDR security profile is based on FAPI 1.0 Advanced security profile. If your client also applies PAR, make sure that client applies RFC 7637 Proof Key for Code Exchange (PKCE) because the Australia CDR security profile requires that you apply PKCE when applying PAR. This can be achieved by using a client profile with the `pkce-enforcer` executor.
```

Reply:
Thank you. I will fix them.